src/tasty/styles/preset.ts

```diff
@@ -10,7 +10,17 @@ function setCSSValue(
   styles[`--${styleName}`] =
     presetName === 'inherit'
       ? 'inherit'
-      : `var(--${presetName}-${styleName}, var(--default-${styleName}, inherit))`;
+      : presetName === 'default'
```

Comment:
I'm sure no one, even you after some period of time will never understand this mess of `?:${}()` symbols. Even if you wanna assign value where it's declared, I would suggest using inline self-calling arrow functions, e.g.

```typescript
styles[`--${styleName}`] = (() => {
  if (presetName === 'inherit') return 'inherit';
  // ...
})()
```

Reply:
Fixed